src/main/java/com/mabl/integration/jenkins/MablStepBuilder.java

```diff
@@ -115,12 +115,12 @@ public void setDisableSslVerification(boolean disableSslVerification) {
 
     @DataBoundSetter
     public void setApiBaseUrl(String apiBaseUrl) {
-        this.apiBaseUrl = apiBaseUrl;
+        this.apiBaseUrl = StringUtils.isEmpty(apiBaseUrl) ? DEFAULT_MABL_API_BASE_URL : apiBaseUrl;
```

Comment:
I am not entirely sure where these get called. Could we prefill form with these values and make fields required. 
Alternatively you can set apiBaseUrl to default in the constructor and not allow set to empty value.

Reply:
This (hidden) field is defined in the config.jelly. If they were not hidden, then they would show up in the configuration form. In other words, apiBaseUrl and appBaseUrl are part of the job configuration. There should not be any need for this normally as the default value in the config.jelly now have the default values. This is just another safety check to avoid setting an empty Url (since you could set the apiBaseUrl to be empty technically in the job configuration xml). There is no way in the configuration to mark a field as mandatory.